cookbook/recipes/common.rb

```diff
@@ -0,0 +1,19 @@
+node.set[:java][:jdk_version] = 7
+
+include_recipe 'java'
+include_recipe 'baragon::build'
+
+%w(/etc/baragon /var/log/baragon).each do |dir|
```

Comment:
There's no best practice that says this :tongue:

Anyway normally I'd agree for a large number of items, especially ones needing sorting, but in this case I think it just adds unnecessary complexity.